### PR TITLE
Resolve #919, #992, #994, #996, #997

### LINKED ISSUE #919: Missing error message when account verify failed

### PR DESCRIPTION
**Pull request by InVincible2016**

### CODE REVIEW

File: src/components/WalletAuthorizationComponent.jsx [hunk 1]
```diff
@@ -196,6 +197,12 @@ export default class WalletAuthorizationComponent extends Component<Props, State
           instruction = 'Waiting for authorization...'
         }
         walletSteps = this.renderMetamaskConnectSteps()
+        if (errors.checkWalletConnection) {
+          errorInstruction = 'MetaMask extension not available'
+        }
+        if (errors.verifyAccount) {
+          errorInstruction = 'MetaMask authorization denied'
```
Comment (timothywangdev) [inline src/components/WalletAuthorizationComponent.jsx]: Wrong metamask account, please switch to the correct one

File: src/components/WalletAuthorizationComponent.jsx [hunk 2]
```diff
@@ -204,6 +211,9 @@ export default class WalletAuthorizationComponent extends Component<Props, State
           instruction = 'Please scan the QR code with MetaMask Mobile app...'
         }
         walletSteps = this.renderMetamaskWalletConnectSteps()
+        if (errors.checkWalletConnection) {
+          errorInstruction = 'MetaMask WalletConnect loading failed'
+        }
         break
```
Comment (timothywangdev) [inline src/components/WalletAuthorizationComponent.jsx]: must consider verify failure, see metamask section

File: src/components/WalletAuthorizationComponent.jsx [hunk 3]
```diff
@@ -212,6 +222,12 @@ export default class WalletAuthorizationComponent extends Component<Props, State
           instruction = 'Please navigate to selected crypto on your Ledger device...'
         }
         walletSteps = this.renderLedgerConnectSteps()
+        if (errors.checkWalletConnection) {
+          errorInstruction = 'Ledger device is not connected'
+        }
+        if (errors.verifyAccount) {
+          errorInstruction = `Ledger ${cryptoType} app is not available`
```
Comment (timothywangdev) [inline src/components/WalletAuthorizationComponent.jsx]: what if ledger address is wrong?

File: src/components/WalletAuthorizationComponent.jsx [hunk 4]
```diff
@@ -223,10 +239,16 @@ export default class WalletAuthorizationComponent extends Component<Props, State
...
-          instruction = 'Creating connection...'
+          instruction = 'Fetching'
         } else if (actionsPending.verifyAccount) {
           instruction = 'Please scan the QR code with Coinbase WalletLink Mobile app...'
         }
+        if (errors.checkWalletConnection) {
+          errorInstruction = 'WalletLink loading failed'
+        }
+        if (errors.verifyAccount) {
+          errorInstruction = `WalletLink verification failed.`
```
Comment (timothywangdev) [inline src/components/WalletAuthorizationComponent.jsx]: better msg: + switch to the correct account?

File: src/components/AddAccountModalComponent.jsx [hunk 5]
```diff
@@ -277,7 +277,6 @@ class AddAccountModalComponent extends Component<Props, State> {
                   id='account name'
                   variant='outlined'
                   label='Account Name'
-                  // disabled={actionsPending.}
                   onChange={event => {
```
[the remark this answers is not in the document]
Comment (InVincible2016) [reply src/components/AddAccountModalComponent.jsx]: This has nothing to do with the proceeding.

File: src/actions/walletActions.js [hunk 6]
```diff
@@ -179,9 +187,21 @@ function newCryptoAccountFromWallet (
...
+  return (dispatch: Function, getState: Function) => {
+    const cryptoAccounts = getState().accountReducer.cryptoAccounts
+    return dispatch({
+      type: 'NEW_CRYPTO_ACCOUNT_FROM_WALLET',
+      payload: _newCryptoAccountFromWallet(name, cryptoType, walletType, cryptoAccounts, options)
+    }).catch(err => {
+      if ((err.msg = 'Account already exists'))
+        dispatch(
+          enqueueSnackbar({
+            message: err.msg,
```
[the remark this answers is not in the document]
Comment (InVincible2016) [reply src/actions/walletActions.js]: This is the right requested error style. black bg with white txt, please try before commenting

**Pull request merged**